crates/subspace-farmer/Cargo.toml

```diff
@@ -97,5 +97,4 @@ binary = [
     "dep:mimalloc",
     "dep:subspace-metrics",
     "dep:supports-color",
-    "dep:tracing-subscriber",
```

Comment:
what abot this one ? Shall we add dep::subspace-logging ?

if so why this one is needed only here and not to binary node ??